src/Iodev/Whois/Punycode/SymfonyPunycodeCovertor.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Iodev\Whois\Punycode;
+
+use Symfony\Polyfill\Intl\Idn\Idn;
+
+class SymfonyPunycodeCovertor implements IPunycode
```

Comment:
This class seems to be removed. Check previous comment please

Reply:
close this thread. Discus in previous thread.